### PR TITLE
TRUS-813 Asset total money amount

### PR DESCRIPTION
**Pull request by codebysachin**
Please review changes for AssetMoneyValue.

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -189,4 +189,8 @@ POST       /chack-agent-details                       controllers.AgentAnswerCon
 #assets
 
 GET        /asset/information-you-need                    controllers.AssetInterruptPageController.onPageLoad()
-POST       /asset/information-you-need                     controllers.AssetInterruptPageController.onSubmit()
\ No newline at end of file
+POST       /asset/information-you-need                     controllers.AssetInterruptPageController.onSubmit()
+GET        /assetMoneyValue                        controllers.AssetMoneyValueController.onPageLoad(mode: Mode = NormalMode)
```
Comment (manishsw) [inline conf/app.routes]: @codebysachin  Update path

File: app/pages/AssetMoneyValuePage.scala [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object AssetMoneyValuePage extends QuestionPage[String] {
+  def unapply(page: Page): Option[Any] = ???
```
Comment (adamconder) [inline app/pages/AssetMoneyValuePage.scala]: Please remove code that is not implemented.

File: app/pages/AssetMoneyValuePage.scala [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+  override def path: JsPath = JsPath \ toString
```
Comment (adamconder) [inline app/pages/AssetMoneyValuePage.scala]: Do we need to scope the path in mongo to `JsPath \ Assets \ toString` where normally we would index this as well? Due to only having 1 monetary amount I think we should index this to conform with logic that @JoshuaH01 started to implement in the page before.

File: test/forms/AssetMoneyValueFormProviderSpec.scala [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    val fieldName = "value"
+
+    behave like fieldThatBindsValidData(
+      form,
+      fieldName,
+      RegexpGen.from(Validation.numericRegex)
+    )
+
+    behave like fieldWithMaxLength(
```
Comment (adamconder) [inline test/forms/AssetMoneyValueFormProviderSpec.scala]: Do we need a test to verify the minimum length of this field?
